Source/DafnyCore/Resolver/PreType/PreTypeConstraints.cs

```diff
@@ -512,6 +525,19 @@ bool TryApplyDefaultAdviceFor(PreTypeProxy proxy) {
       return false;
     }
 
+    bool TryUseCompatibleTypesAsBounds() {
+      // if there is a compatible-types constraint "ty ~~ proxy", then decide on the bound "ty :> proxy"
```

Comment:
What is a compatible types constraint, `ty ~~ proxy` ? If I'm in the code and I read this comment, can I follow links that lead me to a description of these constraints?